tests/Hedgehog.Experimental.Tests/GenTests.fs

```diff
@@ -3,6 +3,7 @@ module Hedgehog.Experimental.Tests.GenTests
 open Xunit
 open Swensen.Unquote
 open Hedgehog
+open Hedgehog
```

Comment:
`open Hedgehog` twice?

Reply:
That's just VF# fooling around. I'll remove it. :)

Reply:
:+1: